### DISCUSSION TITLE
How handle a cancellation of a published VEAT?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#912] [Q&A] [answered]
Hi,
One of our customers wants to know if it is allowed to use a contract award notice to announce the cancellation of a published VEAT. Is that the correct way of doing it? Or should a contract modification notice be published? Any other options to handle this scenario?

Kind regards,
Åsa

**Comment by YvesJo** [comment, marked as answer]
Hi,
There are various options:

- the first one would be to cancel the VEAT notice (making as if this notice did not exist),
- the second option would be to publish a CAN and state that the lot has not been awarded.

A Contract Modification notice can't be applied directly on a VEAT notice as the Contract does not actually exist (While at DAP notice publication, the contract has been prepared, it has not been signed).
